feg/gateway/sbi/sbi.go

```diff
@@ -29,8 +29,10 @@ type SbiServer struct {
 }
 
 func NewSbiServer(listenAddr string) *SbiServer {
+	e := echo.New()
+	e.Use(ServerLoggingMiddleware(nil))
 	return &SbiServer{
-		Server:     echo.New(),
+		Server:     e,
```

Comment:
i think all would be simpler if you were to do

```
func NewSbiServer(listenAddr string) *SbiServer {
	e := echo.New()
	if glog..v == 2 {
		e.Use(ServerLoggingMiddleware(nil))
	}
	return &SbiServer{
		Server:     e,
		ListenAddr: listenAddr,
	}
}
```

Using that you can forget about the logger. Note the log is not dynamic. Either is enabled from start ot it is not